fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -200,15 +201,17 @@
         /** The total number of timeouts that have happened during asyncToSync and their durations. */
         TIMEOUTS("timeouts"),
         /** Total number and duration of commits. */
-        COMMITS("commits")
+        COMMITS("commits"),
+        /** Time for FDB fetches.*/
+        FDB_FETCH_TIME("fdb fetch")
```

Comment:
Also, try to avoid `fdb` in the name of the metric since they are all assumed to be FDB metrics (to allow for further confusion, upstream consumers of these metrics frequently will automatically prefix the metrics with "FDB").  Since I think the point of this is to highlight that it is time and count spent going to the server, maybe it should be:

```
FETCHES("fetches")
```